fe/fe-core/src/main/java/com/starrocks/qe/ConnectContext.java

```diff
@@ -320,6 +324,11 @@ public int getConnectionId() {
 
     public void setConnectionId(int connectionId) {
         this.connectionId = connectionId;
+        this.connectionStartTime = System.currentTimeMillis();
```

Comment:
is this a good way to set startTime in this function?
This may confuse some users

Reply:
You've got a point. I will add a `setConnectionStartTime()` function separately.